src/_includes/layouts/homepage.html

```diff
@@ -79,6 +79,11 @@ <h3 id="{{ event.data.title | slug }}">{{ event.data.title }}</h3>
 					<a href="{{ event.url }}" aria-label="View Details for {{ event.data.title }}">View Details &#8250;</a>
 				</article>
 				{% endfor %}
+
+				<a href="/sesh" class="event-item" id="add-event-link">
+					{% include partials/add.svg %}
+					<span>* Insert Your Awesome Event Idea Here *</span>
```

Comment:
I'm noticing when I pull up the page that the last asterisk is dropping down to its own line. I think maybe we should remove the spaces between the asterisks and the word.